Add Ceres products to SORA docs (#225)
sora-xor/sora-docs: src/ceres/launchpad.md

---
title: "Launchpad | SORA Docs"
head:
  - - meta
    - name: description
      content: "Ceres Launchpad is the first station for new projects on the SORA network. "
  - - meta
    - name: keywords
      content: "Ceres, DeFi, launchpad, Polkaswap, SORA network"
---

# Ceres Launchpad

> [Ceres Launchpad](https://dapps.cerestoken.io/launchpad) is the first station for new projects on the SORA network. 

Reviewer: ```suggestion
The [Ceres Launchpad](https://dapps.cerestoken.io/launchpad) is the first station for new projects on the SORA network. 
```

> With Launchpad, new projects can easily launch their tokens on Polkaswap and users will get more security because Ceres Liquidity Locker

Reviewer: ```suggestion
With Launchpad, new projects can easily launch their tokens on Polkaswap, and users have added security and peace of mind thanks to the Ceres Liquidity Locker, preventing pump and dump schemes.
```

> and Ceres Token Locker are integrated into Launchpad.

Reviewer: ```suggestion
The Ceres [Liquidity Locker](/liquidity-locker.md) and [Token Locker](/token-locker.md) are integrated into the Launchpad.
```

> The Ceres [Liquidity Locker](/liquidity-locker.md) and [Token Locker](/token-locker.md) are integrated into the Launchpad.

Reviewer: ```suggestion
The Ceres [Liquidity Locker](./liquidity-locker.md) and [Token Locker](./token-locker.md) are integrated into the Launchpad.
```

Reviewer: pls commit this urgent @DanijelCBS ! 
Cheers

Reviewer 2: Fixed
